ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -393,11 +393,22 @@ export default defineComponent({
       })
     })
 
+    const cleanUpInput = (dirtyValue: string|undefined) => {
+      if (dirtyValue === undefined) {
+        return undefined
+      }
+      return dirtyValue
+        .trim()
+        .replaceAll(/[\u200B-\u200D\uFEFF\u200E\u200F]|(?:&#x200E;)/g, '')
+        .replaceAll(/[\u2018\u2019]/g, "'")
+        .replaceAll(/[\u201C\u201D]/g, '"')
+    }
+
```

Comment:
Judging by the regex this will clean hidden characters as well?

Could you add a docblock comment to describe the function definition of what it will clean for those unfamiliar with reading regex?